packer/maps.go

```diff
@@ -0,0 +1,74 @@
+package packer
+
+import (
+	"fmt"
+)
+
+type MapOfProvisioner map[string]func() (Provisioner, error)
+
+func (mop MapOfProvisioner) Has(provisioner string) bool {
+	_, res := mop[provisioner]
+	return res
+}
+
+func (mop MapOfProvisioner) Start(provisioner string) (Provisioner, error) {
+	p, found := mop[provisioner]
+	if !found {
+		return nil, fmt.Errorf("Unknown provisioner %s", provisioner)
+	}
+	return p()
+}
+
+func (mop MapOfProvisioner) List() []string {
+	res := []string{}
+	for k := range mop {
+		res = append(res, k)
+	}
+	return res
+}
+
+type MapOfPostProcessor map[string]func() (PostProcessor, error)
+
+func (mopp MapOfPostProcessor) Has(provisioner string) bool {
+	_, res := mopp[provisioner]
+	return res
+}
+
+func (mopp MapOfPostProcessor) Start(provisioner string) (PostProcessor, error) {
+	p, found := mopp[provisioner]
```

Comment:
here too. i think this was just copypaste and has no functinal problems but I think it'll be less confusing to make sure the variable names are correct.

Reply:
yup you are right; I copy pasted these around a bit and skipped this !